vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue

```diff
@@ -213,14 +208,25 @@ export default {
       const ownerNumber = this.ownerNumber;
       const listingPrice = this.price;
       const listingState = this.listingState;
-      const propertyType = this.propertyType;
+      const propertyType = this.type;
+      const totalBeds = this.beds; 
+      const totalBedrooms = this.bedrooms; 
+      const totalBaths = this.bathrooms; 
+      const hasWifi = this.wifiCheckbox;
+      const hasAC = this.acCheckbox;
+      const hasWasherDryer = this.washerCheckbox;
+      const hasGym = this.gymCheckbox;
+      const hasPool = this.poolCheckbox;
+      const hasParking = this.parkingCheckbox;
+      const hasGreenspace = this.greenspaceCheckbox;
 
-      
+      // Create Listing Object
       const currentListing = {
         name: listingTitle,
         price: listingPrice,
         type: propertyType,
         desc: listingDescription,
+
         images: [
           "https://cdngeneral.rentcafe.com/dmslivecafe/3/1104500/METRO%20GATEWAY%20IMG%2003(2).jpg?crop=(0,0,300,191.25000000000028)&cropxunits=300&cropyunits=200&quality=85&scale=both",
```

Comment:
where are these images from?

Reply:
Those are temp images for now (placeholders)